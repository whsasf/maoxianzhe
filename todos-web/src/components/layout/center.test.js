import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Center from "./center";

it("should renders correctly", () => {
  const tree = renderer.create(<Center>somthing</Center>).toJSON();
  expect(tree).toMatchSnapshot();
});
