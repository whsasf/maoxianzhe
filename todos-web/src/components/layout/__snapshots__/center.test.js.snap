// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
.c0 {
  width: 100%;
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div
  className="c0"
>
  somthing
</div>
`;
